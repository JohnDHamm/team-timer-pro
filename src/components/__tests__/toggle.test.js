import React from 'react'
import renderer from 'react-test-renderer'
// import Toggle from '../toggle'

describe('<Toggle />', () => {
  const labels = ["left", "right"];
  // jest.mock('expo-haptics', () => 'Haptics');

  // test('it renders correctly', () => {
  //   const tree = renderer.create(<Toggle labels={labels} selected={0} />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });

  test('test placeholder', () => {
    expect(true).toBe(true)
  })
});
