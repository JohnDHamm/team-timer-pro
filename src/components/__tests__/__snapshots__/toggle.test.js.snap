// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toggle /> it renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#BADA55",
        "borderBottomLeftRadius": 3,
        "borderColor": "#BADA55",
        "borderTopLeftRadius": 3,
        "borderWidth": 1,
        "opacity": 1,
        "paddingBottom": 7,
        "paddingHorizontal": 13,
        "paddingTop": 3,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "dosis-regular",
            "fontSize": 25,
          },
          Object {
            "color": "#6E5EAB",
          },
        ]
      }
    >
      left
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderBottomRightRadius": 3,
        "borderColor": "#BADA55",
        "borderTopRightRadius": 3,
        "borderWidth": 1,
        "opacity": 1,
        "paddingBottom": 7,
        "paddingHorizontal": 13,
        "paddingTop": 3,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "dosis-regular",
            "fontSize": 25,
          },
          Object {
            "color": "#CCCCCC",
          },
        ]
      }
    >
      right
    </Text>
  </View>
</View>
`;
